WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -108,6 +108,10 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	[JsonConverter(typeof(MoneyBtcJsonConverter))]
 	public Money MaxSuggestedAmountBase { get; set; } = Money.Coins(0.1m);
 
+	[DefaultValue(1)]
+	[JsonProperty(PropertyName = "ParallelRounds", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public int ParallelRounds { get; set; } = 1;
```

Comment:
Shouldn't this be `2`?

`var registrableRoundCount = 1`
`int roundsToCreate = 1 - 1 = 0;`
`for (int i = 0; i < roundsToCreate; i++) { ... }` <= won't create another round.